typescript/lib/services/service_context.ts

```diff
@@ -0,0 +1,129 @@
+import * as bindingsJsGrpc from '../../kurtosis_core_rpc_api_bindings/api_container_service_grpc_pb'; //TODO - should I be importing all modules or just specific ones for best practices (need to check)
```

Comment:
I personally prefer explicitness (so `import { Module1, Module2 } from `./the_file`), but once you get beyond a certain number of imports I think Visual Studio will probably condense it into a `import *` anyways